app/javascript/gobierto_data/webapp/components/commons/DownloadButton.vue

```diff
@@ -5,6 +5,7 @@
       :text="labelDownloadData"
       icon="download"
       color="var(--color-base)"
+      icon-color="rgba(var(--color-base-string), .5)"
```

Comment:
Just one question: if the common button color (both) is the `base`, wouldn't be better to set as default fallback in the `Button` component? In that way, whenever we wanna use it and do not need an additional color, we don't need to pass such props

Reply:
Yes. Good point, as usual. So:

```
color: {
      type: String,
      default: 'var(--color-base)'
    }

iconColor: {
      type: String,
      default: 'var(--color-base)'
    },
```